### ISSUE TITLE
Reproduce classification fine-tuned results on ModelNet40

### ISSUE THREAD

**Issue by fishcrap** [SimingYan/IAE#15]
Hi,  thanks for your great work. I try to fine-tune DGCNN with the given pretrained model(`modelnet40_clsft.pt`) on ModelNet40. However, even if I follow the suggestions in #8, setting `batch_size=32, k=40, num_points=2048, epoch=250`, the OA still cannot reach 94.2, which gives 93.4. By further comparing your released log with mine, I find the configuration of `data_aug, scheduler` is different. I set `data_aug=1` and `scheduler=cos` following your log, and conduct the experiments again for twice, which gives OA = 93.6 and 93.5 respectively. The results are still different from the paper-reported 94.2 OA. 

Meanwhile, I test DGCNN with the pretrained weights(`modelnet40_trained.pt`), but it gives 94.0 OA and 91.1 Mean Class Acc, which is different from 94.2 OA and 91.6 Mean Class Acc in your released log.

Could you please give me some advice on reproducing the classfication results? Appreciate for your help!

**Comment by SimingYan** [comment]
Hi,

Could you please share your coding environment? e.g., conda version, pytorch version, python version.

**Comment by fishcrap** [comment]
您好！您的邮件我已收到，我会尽快处理的，谢谢！顺颂时祺！

**Comment by fishcrap** [comment]
My emvironment is bit different from yours as I met some difficulty in setting up the project when just following the suggestions in README. So I choose the environment I usually use. Here is my coding environment:
```
conda                     	4.10.3
pytorch                   	1.9.0
python                    	3.7.11
torch-scatter             	2.0.7
...
cudatoolkit			11.1.74
``` 
I run the project on Ubuntu 18.04 with GeForce RTX 3090.

Does the coding environment have a big impact on the performance, leading to the acc gap?  Thanks for your help!

**Comment by SimingYan** [comment]
I'm not sure how big influence will be for the fine-tuning procedure, but at least under the same environment, our pretrained weight should show the same result. So my suggestion is training under the same setting.

**Comment by fishcrap** [comment]
Sorry for the late reply. :sob:

Thanks for your suggestion! Undering the same setting on Ubuntu 18.04 with Tesla V100, the pretrained weights for test now show the same result as the released log.

However, for the procedure of fine-tuning, I have trained for four times, which still gives $OA = 93.5 (\pm0.1)$. Could you please give me some suggestions on the fine-tuning?

**Comment by SimingYan** [comment]
That's a little weird. Even though I agree that training on modelnet40 is not stable, I think it should still can get at least 93.8. Maybe you can print out the OA of each class and make a comparison between our released weight and your weight.

**Comment by fishcrap** [comment]
Many thanks for your prompt reply! :heart:

I will try it later.
